iocore/net/QUICNetVConnection.cc

```diff
@@ -266,6 +277,12 @@ QUICNetVConnection::start()
   this->_frame_dispatcher->add_handler(this->_stream_manager);
   this->_frame_dispatcher->add_handler(this->_path_validator);
   this->_frame_dispatcher->add_handler(this->_handshake_handler);
+
+  if (this->thread) {
```

Comment:
Is this check for client side? I have no idea why some NetVC don't have any thread.

Reply:
On test ? or other case which event system not initialize completely.

Reply:
Hmm, I think we should not make such a situation happen. Behaving differently on tests doesn't make sense. And if it was not a test, we should definitely set a thread.
Did you find any actual cases that require this check while you are working on this?

Reply:
No. Ok I'll remove it and add assertion here.